src/client/sound/sound_singleton.cpp

```diff
@@ -70,4 +81,37 @@ SoundManagerSingleton::~SoundManagerSingleton()
 	infostream << "Audio: Global Deinitialized." << std::endl;
 }
 
+static std::vector<ALCint> init_attrs()
+{
+	std::vector<ALCint> attrs;
+
+	s32 sound_sources_mono = -1;
+	if (!g_settings->getS32NoEx("sound_sources_mono", sound_sources_mono))
+		sound_sources_mono = -1;
```

Comment:
It always exists, because of the setting default.
```suggestion
	s32 sound_sources_mono = g_settings->getS32("sound_sources_mono", sound_sources_mono);
```